develop/application-devs/system-parachains/interact-asset-hub.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Interacting with Asset Hub
+description: A general page which describes what Asset Hub is, its capabilities, and how it can be utilized in the context of building on the wider Polkadot ecosystem.
+---
+
+The relay chain doesn't natively support assets beyond its native token, such as DOT.  This functionality exists in parachains. On both Polkadot and Kusama, this parachain is called *Asset Hub*.
+
+The Asset Hub provides a first-class interface for creating, managing, and using fungible and
+non-fungible assets. The fungible interface is similar to Ethereum's ERC-20 standard. However, the
+data structures and stateful operations are encoded directly into the chain's runtime, making
+operations fast and fee-efficient.
+
+Beyond merely supporting assets, integrating an Asset Hub into your systems has several benefits for
+infrastructure providers and users:
+
+- Support for on-chain assets
+- Significantly lower transaction fees (about 1/10) than the relay chain
+- Significantly lower deposits (1/100) than the relay chain. This includes the existential deposit
+  and deposits for proxy/multisig operations
+- Ability to pay transaction fees in certain assets. As in, accounts would **not** need DOT to exist
+  on-chain or pay fees
+
+The Asset Hub will use DOT as its native currency. Users can transfer DOT from the relay chain into
+the Asset Hub and use it natively. The relay chain will also accept DOT transfers from the Asset Hub
+back to the relay chain for staking, governance, or any other activity.
+
+Using the Asset Hub for DOT/KSM balance transfers will be much more efficient than the relay chain
+and is highly recommended. Until domain-specific parachains are built, the relay chain will still
+need to be used for staking and governance.
+
+## Assets Basics
+
+See the [assets pallet](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame/assets){target=\_blank} for
+the most up-to-date info and reference documentation.
+
+Assets are stored as a map from an ID to information about the asset, including a management team,
```

Comment:
I'd add a small clarification that this only applies to Local Assets (since Foreign assets are stored based on MultiLocation instead of integer IDs)